estate/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Real Estate',
+    'depends': [
+        'base'
+    ],
+    'category': 'Tutorials/Real Estate',
+    'data': [
+        'security/ir.model.access.csv',
+
+        'views/estate_property_views.xml',
+        'views/estate_menus.xml'
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False
+}
```

Comment:
You should add an empty line at the end of each file. This can be configured in your IDE such that every time you save a file, it automatically adds the empty line at the bottom. This comment also applies to the other files